### ISSUE TITLE
Avoid referring to Markdown when formatting messages

### ISSUE THREAD

**Issue by tripu** [w3c/mailing-list-archives#28]
Markdown was inspired by pseudo-formatting in mail messages, yes. But people don't write their e-mails in Markdown on purpose. Some conventions in e-mail (like _&#42;emphasis&#42;_, _&gt; quotations_ and _&#42; bullet lists_) work as expected by Markdown, but others don't.

Don't mention MD on the settings, and don't show its logo&nbsp;&mdash;&nbsp;it's misleading, and many users won't even know what it is.

**Comment by gosko** [comment]
I don't think we should get too fancy with trying to format messages nicely -- email formats and practices vary a lot so it will be impossible to get it exactly right all of the time, but it needs to do a good job 99%+ of the time to avoid being a nuisance.

Including an intelligent and convenient way to show/hide quoted text is more important in my opinion (though that will also be difficult to get exactly right). Maybe we can take cues from places like gmail, and @r12a's thread flattener.

**Comment by r12a** [comment]
There be dragons here.  While working on my thread flattener i looked at a whole bunch of other systems.  They all did clever things to format email, but none of them actually pulled it off.  That's why i went for the simple approach i did, but it seems to produce useful results.

For example, a typical trick involved reassembling lines into paragraphs, but that always ran foul of the fact that many email authors intentionally use the plain text approach to create things like tables, or special layouts, which then get screwed up by the supposedly clever algorithms which actually don't have a clue what the user actually intended.

**Comment by gosko** [comment]
Yes, it seems almost impossible to do a good job of this.

Whatever we come up with, it should be easy and obvious for users how to toggle it on/off (both for smart formatting and quoted text hiding/display)

**Comment by tripu** [comment]
> Whatever we come up with, it should be easy and obvious for users how to toggle it on/off.

Agreed. That's what I wanted to do with [my early proposal](https://w3c.github.io/mailing-list-archives/samples/message-proposal-markdown.html): easy to disable and (with the help of #18), once it's disabled it won't bother the user again.

Thanks for those warnings, @r12a!
